src/components/Buttons/Button/Button.tsx

```diff
@@ -1,12 +1,32 @@
-import React, { forwardRef } from 'react';
-import PropTypes from 'prop-types';
+import React from 'react';
 import bem from '../../../utils/bem';
 import styles from './Button.scss';
-import { CONTEXTS, SIZES } from '../../../constants';
+import { ButtonTypes, ContextTypes, SizesTypes } from '../../../constants';
+
+// Any other attributes (onClick, onFocus etc.) are
+// supported although not defined in propTypes
+interface Props extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'size'> {
+    /** The label of the button */
+    children: [string, Node];
```

Comment:
`[string, Node]` declares a tuple that accepts string as a first element and Node as second.
Syntax of "OneOf" logic is `A | B`.

Also, `Node` is not `ReactNode`, it's a [DOM Node](https://developer.mozilla.org/en-US/docs/Web/API/Node)

Should be
```
children: React.ReactNode;
```

This applies to all the other places where `Node` is being used.